site/components/map/ServicesMap.tsx

```diff
@@ -13,21 +13,22 @@ import {
     useMemo,
```

Comment:
I've noticed a couple of issues with the functionality here:

- Hover on unselected stops not showing the tooltip (on stops and services maps)
- Services aren't being added to the list of selected services on clicking select all, stops are added but not services

Reply:
The first one I was told to implement it that way for those stops in the polygon that are grey

Reply:
fixed the services issue